recipes/websockify/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "websockify" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/novnc/websockify/archive/v{{ version }}.tar.gz
+  sha256: 6ebfec791dd78be6584fb5fe3bc27f02af54501beddf8457368699f571de13ae
+
+build:
+  # Independent of Python version, but rebind.so depends on OS
+  number: 0
+  entry_points:
+    - websockify = websockify.websocketproxy:websockify_init
+  script: "{{ PYTHON }} -m pip install . -vv"  # [osx or win]
+  # Use build.sh for Linux
+  missing_dso_whitelist:
+    - /lib64/libc.so.6
+    - /lib64/libdl.so.2
+
+requirements:
+  host:
+    - {{ compiler('c') }}
```

Comment:
This like should be in a `build` section under `requirements`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L38-L41